packages/fhir-router/src/graphql/graphql.test.ts

```diff
@@ -743,6 +762,13 @@ describe('GraphQL', () => {
                     resource {
                       ...on ServiceRequest {
                         id
+                        asNeededCodeableConcept {
+                          coding {
+                            extension {
+                              url
```

Comment:
Out of curiosity, do we count these simple object traversals as "depth"?  I.e., navigating into a `BackboneElement`?

I'm realizing that we've been talking about "depth" in general, but there's a big difference between getting properties off of a nested `BackboneElement` vs resolving by searching.

Reply:
Yep, it's currently counted simply as each nested field counting as an additional layer of depth